### ISSUE TITLE
More accessibility issues

### ISSUE THREAD

**Issue by cami-dev** [TheIdentitySelector/thiss-js#309]
Hello! Thank you for fixing many of the previous accessibility problems in the earlier issue https://github.com/TheIdentitySelector/thiss-js/issues/307
We have tested through the pages after the update and now it almost passes :) great work! We have some small fixes we would love if you could have a look at to comply with our accessibility standards. If you would like to connect to discuss anything of the things below don't hesitate to send me a message.

Page - Find your institution
1. No visible focus indicator on the input field to search for institutions when using a keyboard. Atm the field looks the same when it has focus and when it does not. Would be great to add a border or something when it has focus. See WCAG rule: https://www.w3.org/WAI/WCAG22/Understanding/focus-visible.html

2. The aria-live for the search announcement now works but it's very talkative. It reads all matches shown and that becomes a lot for a screen reader user so if possible here, it would be better to add a separate div which only contains the number of hits or if no hits were found.
Ex. 
```
<div class="results search-list" id="resultwidget" aria-live="assertive">
	@if(toomany)
	<div id="ds-search-header">
		<div id="too-many-results-notice" class="type-ahead-alert">
			<div role="alert">
				<p><span class="bold">34 <span data-i18n="ds-too-many-result-matches">Matches</span></span> <span data-i18n="ds-too-many-result-keep-typing">keep typing to refine your search</span></p>
				<p><a id="showall" href="#" data-i18n="ds-too-many-result-show">Show me all matches anyway</a></p>
			</div>
		</div>
	</div>
	 @else
	<div class="sr-only">Showing 10 matches</div>
</div>
<!--Leave the ul outside of the aria-live so the user can choose if it wants to read it all -->
<ul id="ds-search-list" class=""></ul>

(Not sure if you already have a screen-reader only class (sr-only) if not here is an example https://v1.tailwindcss.com/docs/screen-readers it hides it from the visible eye but screen readers can read it)

```

3. The about us link in the footer goes to a broken page. https://seamlessaccess.org/about/ does not seem to exist? 
<img width="1428" height="652" alt="Image" src="https://github.com/user-attachments/assets/9849f3f2-dde9-45af-bdb1-8e90736db7c4" />

4. The button to remove institutions under edit should be a button element to be accessible for all instead of a div with a click listener. Just style a button to look like the div does now. 
<img width="1429" height="521" alt="Image" src="https://github.com/user-attachments/assets/f45c2c1e-965f-4e82-b8c5-69e44c29b840" />

5. Add info of which element you are deleting when editing Institutions. The aria-label would need to have a dynamic value like aria-label="`Remove Institution ${currentSelectedInstitution} from remembered institutions`"
<img width="1443" height="577" alt="Image" src="https://github.com/user-attachments/assets/ca9c2752-12fc-4daa-86fc-895e224d1a71" />

6. The blue links at the bottom menu does not have enough contrast ratio to the background to pass WCAG: https://www.w3.org/WAI/WCAG21/Understanding/contrast-minimum

7. The page logo now has no alt attribute at all which gives an error for screen readers. We suggested to leave it blank not remove it completely. So alt=" ". See https://www.w3.org/WAI/tutorials/images/decorative/

**Comment by Bojhan** [comment]
Ah, your right about 6. I think I missed that reviewing. @enriquepablo we can use secondary-blue

**Comment by enriquepablo** [comment]
I hae addresses all issues consigned her. Thanks for the report!

**Comment by cami-dev** [comment]
I am a bit confused @enriquepablo I can only see you changed the color of the link nothing more? We would really like all of the above fixed for it to follow the laws for accessibility in EU.

**Comment by enriquepablo** [comment]
You are right @cami-dev - there was a confusion with #307 . Reopening and fixing the rest of the findings.
